[2024-10-24T15:46:56.851+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: random_number_dag2.generate_random_number2 manual__2024-10-24T15:46:41.020300+00:00 [queued]>
[2024-10-24T15:46:56.872+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: random_number_dag2.generate_random_number2 manual__2024-10-24T15:46:41.020300+00:00 [queued]>
[2024-10-24T15:46:56.872+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-10-24T15:46:56.897+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): generate_random_number2> on 2024-10-24 15:46:41.020300+00:00
[2024-10-24T15:46:56.901+0000] {standard_task_runner.py:57} INFO - Started process 678 to run task
[2024-10-24T15:46:56.905+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'random_number_dag2', 'generate_random_number2', 'manual__2024-10-24T15:46:41.020300+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/random_number_dag2.py', '--cfg-path', '/tmp/tmpyv5vhs16']
[2024-10-24T15:46:56.907+0000] {standard_task_runner.py:85} INFO - Job 79: Subtask generate_random_number2
[2024-10-24T15:46:56.972+0000] {task_command.py:410} INFO - Running <TaskInstance: random_number_dag2.generate_random_number2 manual__2024-10-24T15:46:41.020300+00:00 [running]> on host 29df6f23ff28
[2024-10-24T15:46:57.117+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='random_number_dag2' AIRFLOW_CTX_TASK_ID='generate_random_number2' AIRFLOW_CTX_EXECUTION_DATE='2024-10-24T15:46:41.020300+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-24T15:46:41.020300+00:00'
[2024-10-24T15:46:57.118+0000] {logging_mixin.py:150} INFO - Исходное число: 65, Результат: 4225
[2024-10-24T15:46:57.118+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-10-24T15:46:57.133+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=random_number_dag2, task_id=generate_random_number2, execution_date=20241024T154641, start_date=20241024T154656, end_date=20241024T154657
[2024-10-24T15:46:57.215+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-10-24T15:46:57.269+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
